README.md

```diff
@@ -49,16 +85,211 @@ grey_fg2 = 20% lighter than grey
+  pink = "",
+  line = "",
+  green = "",
+  vibrant_green = "",
+  nord_blue = "",
+  blue = "",
+  seablue = "",
+  yellow = "",
+  sun = "",
+  purple = "",
+  dark_purple = "",
+  teal = "",
+  orange = "",
+  cyan = "",
+  statusline_bg = "",
+  lightbg = "",
+  pmenu_bg = "",
+  folder_bg = ""
+}
+-- syntax
+M.base_16 = {
+  base00 = "",
+  base01 = "",
+  base02 = "",
+  base03 = "",
+  base04 = "",
+  base05 = "",
+  base06 = "",
+  base07 = "",
+  base08 = "",
+  base09 = "",
+  base0A = "",
+  base0B = "",
+  base0C = "",
+  base0D = "",
+  base0E = "",
+  base0F = ""
+}
+-- overriding highlights (see #overriding-a-highlight)
+M.polish_hl = {}
+
+-- set the theme type whether is dark or light
+M.type = "<DARK/LIGHT>"
+
+-- assinging the theme
+M.require("base46").override_theme(M, "<YOUR THEME NAME>")
+
+-- don't return the empty highlighting this will cause an error since those will used to generate the highlightings
+return M
 ```
+
+## Override a highlight
+
+- `M.Polish` used to override highlights since `base46` will generate them
+- to override a highlight you should specify the highlight
+
+```lua
+M.polish_hl = {
+ ["@keyword"] = { bold = true}
+ String = { italic = false }
+}
+```
+
+## Setting highlightings
+
+in `base46` you can set the highlighting in diffrent way
+
+```lua
+-- ...
+M.base_30 = {
+-- ...
+ white = "#FFFFFF",
+ yellow = "yellow",
+-- ...
+}
+M.base_16 = {
+-- ...
+ base00 = "#000000",
+ base05 = "blue"
+-- ...
+}
+-- ...
+```
+
+### Base46 table options
+
+- specify the the highlightigs options for each highilght
+- you can set these options to override highlightings only in `M.polish_hl`
+  property
+
+| highlighting    | Role                                                               | Type      | Value                                       |
+| --------------- | ------------------------------------------------------------------ | --------- | ------------------------------------------- |
+| `fg`            | foreground                                                         | `string`  | color name or hex code or "none"            |
+| `bg`            | background                                                         | `string`  | color name or hex code or "none"            |
+| `sp`            | color of underlines. (more info `:h guisp`)                        | `string`  | color name or hex code or "none"            |
+| `link`          | link a highlight to another highlight options                      | `string`  | highlight name or "none"                    |
+| `italic`        | text style                                                         | `boolean` | boolean value                               |
+| `bold`          | text style                                                         | `boolean` | boolean value                               |
+| `blend`         | combining colors to create a new color (not often used in nvchad)  | `number`  | integer between 0 and 100, level of opacity |
```

Comment:
type here is integer, actually

And this only takes effect for floating windows